src/mantine-core/src/Floating/FloatingArrow/get-arrow-position-styles.ts

```diff
@@ -1,47 +1,6 @@
 import { CSSObject } from '@mantine/styles';
 import type { FloatingPosition, FloatingSide, FloatingPlacement } from '../types';
 
-function horizontalSide(
-  placement: FloatingPlacement | 'center',
-  arrowY: number,
-  arrowOffset: number
-) {
-  if (placement === 'center') {
-    return { top: arrowY };
-  }
-
-  if (placement === 'end') {
-    return { bottom: arrowOffset };
-  }
-
-  if (placement === 'start') {
-    return { top: arrowOffset };
-  }
-
-  return {};
-}
-
-function verticalSide(
-  placement: FloatingPlacement | 'center',
-  arrowX: number,
-  arrowOffset: number,
-  dir: 'rtl' | 'ltr'
-) {
-  if (placement === 'center') {
-    return { [dir === 'ltr' ? 'left' : 'right']: arrowX };
-  }
-
-  if (placement === 'end') {
-    return { [dir === 'ltr' ? 'right' : 'left']: arrowOffset };
-  }
-
-  if (placement === 'start') {
-    return { [dir === 'ltr' ? 'left' : 'right']: arrowOffset };
-  }
-
-  return {};
-}
```

Comment:
Those two functions are not needed anymore, as position is correctly handled by Floating UI, including RTL. So we can directly set `left` for vertical and `top` for horizontal sides